configure.ac

```diff
@@ -226,18 +226,29 @@ AC_SUBST([JASMINE_REPORT_ARGUMENT])
 PKG_CHECK_MODULES([EOS_SDK], [
 	$EOS_REQUIRED_MODULES
 	$EOS_REQUIRED_MODULES_PRIVATE])
-PKG_CHECK_MODULES([WEBHELPER2_EXTENSION], [
-	$GLIB_REQUIREMENT
-	$GOBJECT_REQUIREMENT
-	$WEBKIT2_REQUIREMENT])
-PKG_CHECK_MODULES([WEBHELPER2_PRIVATE], [
-	$GLIB_REQUIREMENT
-	$WEBKIT2_REQUIREMENT])
 
 # Check installed GIRs for webhelper JS module
 EOS_CHECK_GJS_GIR([GLib], [2.0])
-EOS_CHECK_GJS_GIR([WebKit], [3.0])
-EOS_CHECK_GJS_GIR([WebKit2], [4.0])
+
+# Webhelper checks
+AC_ARG_ENABLE([webhelper],
+              [AS_HELP_STRING([--disable-webhelper], [Disable webhelper build])],
+              [],
```

Comment:
nitpick: The Autotools Mythbuster recommends as a best practice to leave out the optional arguments to `AC_ARG_ENABLE` and, for a default-enabled argument, make all the checks (`AS_IF`, `AM_CONDITIONAL`) work like `test "x$enable_webhelper" != xno`.